sdks/python/apache_beam/io/filebasedsource_test.py

```diff
@@ -87,6 +87,22 @@ class EOL(object):
 def write_data(
     num_lines, no_data=False, directory=None, prefix=tempfile.template,
     eol=EOL.LF):
+  """Writes test data to a temporary file.
+
+  Args:
+    num_lines (int): The number of lines to write.
+    no_data (bool): If :data:`True`, empty lines will be written, otherwise
+      each line will contain a concatenation of b'line' and the line number.
+    directory (str): The name of the directory to create the temporary file in.
+    prefix (str): The prefix to use for the temporary file.
+    eol (int): The line ending to use when writing.
+      :class:`~apache_beam.io.textio_test.EOL` exposes attributes that can be
+      used here to define the eol.
+
+  Returns:
+    Tuple[str, List[str]): A tuple of the filename and a list of the written
```

Comment:
Thanks. I wish we had more docstrings like this one throughout SDK internals.

Tuple[str, List[bytes]] in Line 103?

Reply:
You're right. Fixed it.